OptimizelySDK/Data Model/DispatchEvents/EventForDispatch.swift

```diff
@@ -35,7 +35,13 @@ import Foundation
 }
 
 extension EventForDispatch {
+    
     override public var description: String {
         return "[url] \(url) (" + (String(data: body, encoding: .utf8) ?? "UNKNOWN") + ")"
     }
+    
```

Comment:
why did you move this?

Reply:
Looks like redundant to "isEqual" above. WIl be removed.